code/pizzabot.js

```diff
@@ -1,7 +1,98 @@
-const vegetarian = "Vegetarian Pizza"
-const hawaiian = "Hawaiian Pizza"
-const pepperoni = "Pepperoni Pizza"
+// 1. Definition of variables 
 
+const vegetarian = "Vegetarian"
+const hawaiian = "Hawaiian"
+const pepperoni = "Pepperoni"
 const pizzaPrice = 80
 
-//Put your Javscript code here:
+
+// 2. Functions
+
+// Function for validating ordername
+const validateOrderName = (orderName) => {
+    if (orderName.toLowerCase() === vegetarian.toLowerCase()) {
+        return true;
+    } else if (orderName.toLowerCase() === hawaiian.toLowerCase()) {
+        return true;
+    } else if (orderName.toLowerCase() === pepperoni.toLowerCase()) {
+        return true;
+    } else {
+        alert('Select a pizza from the menu!');
+        return false;
+    }
+};
+
+// Function for calculating total cost
+const calculateTotalCost = (orderQuantity, pizzaPrice) => {
+    let orderTotal = orderQuantity * pizzaPrice;
+    return orderTotal;
+};
+
+// Function for calculating total cooking time
+const calculateCookingTime = (orderQuantity) => {
+    let orderTime = 0; 
+    if (orderQuantity <= 2) {
+        orderTime = 10;
+        return orderTime;
+
+    } else if (orderQuantity < 6) {
+        orderTime = 15;
+        return orderTime;
+
+    } else {
+        orderTime = 20;
+        return orderTime;
+    }
+};
+
+//Function for asking what kind of pizza user would like to order 
+const askPizzas = () => {
+    const orderName = `On our menu we have ${vegetarian}, ${hawaiian} and ${pepperoni}. Choose which pizza you want.`
+    document.getElementById("pizzaName").innerHTML = orderName;
```

Comment:
You seem to use semicolons only when you want. We have to be consequent - either use them everywhere or don't use them at all